nemo_text_processing/text_normalization/ja/taggers/cardinal.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pynini
+from pynini.lib import pynutil
+
+from nemo_text_processing.text_normalization.ja.graph_utils import NEMO_DIGIT, GraphFst
+from nemo_text_processing.text_normalization.ja.utils import get_abs_path
+
+
+class CardinalFst(GraphFst):
+    """
+    Finite state transducer for classifying cardinals
+        e.g. 23 -> cardinal { integer: "二十三" } 
+    """
+
+    def __init__(self, deterministic: bool = True):
+        super().__init__(name="cardinal", kind="classify", deterministic=deterministic)
+
+        graph_zero = pynini.string_file(get_abs_path("data/numbers/zero.tsv"))
+        graph_digit = pynini.string_file(get_abs_path("data/numbers/digit.tsv"))
+        # graph_digit_alt = pynini.string_file(get_abs_path("data/numbers/digit_alt.tsv"))

```

Comment:
If the commented lines are intended for developers to comment/uncomment, it's good to have that mentioned somewhere in the file. Otherwise, if they'd don't bring anything to the table, I'd just remove them.